CHANGELOG.md

```diff
@@ -143,7 +152,8 @@ a long list of states to go through.
 * Support for data in states
 * Change log added
 
-[Unreleased]: https://github.com/korken89/smlang-rs/compare/v0.5.1...master
+[Unreleased]: https://github.com/korken89/smlang-rs/compare/v0.6.0...master
+[v0.6.0]: https://github.com/korken89/smlang-rs/compare/v0.5.1...v0.6.0
```

Comment:
The bot complains about broken links every night. Could we fix those? There should be issues around what exactly has an issue.

Reply:
Those issues appear to be caused by the fact that the `v0.3.4` and `v0.3.5` tags don't exist in Github, but are referenced in the CHANGELOG. I'll figure out the commits and tag them.

Reply:
Thanks!